benchmarks/tests/test_monitor.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+from benchmarks.utils import monitor, repr_func  # replace with the actual import path
```

Comment:
Out of curiosity: is this supposed to work without installation? I'm a bit surprised that it does.

Reply:
Yes, it does work, since there is an `__init__` file in `benchmarks/` :)